components/empty-state/getIllustration.js

```diff
@@ -0,0 +1,25 @@
+import { html } from 'lit';
+import { loadSvg } from '../../generated/empty-state/presetIllustrationLoader.js';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+
+function fixSvg(svgStr) {
```

Comment:
No, that's not necessary. It's already private to this module since it's not being exported. In components we prefix private members with `_` because the component is effectively public.

Reply:
Is this `fixSvg` function necessary? For icons, it does a few things that aren't necessary here.

Reply:
Nope, it doesn't seem necessary, thanks.

Reply:
Cool... that might eliminate the need for this module altogether. You may want to check what happens when the consumer provides an invalid illustration name.

Reply:
Yeah, I was trying to avoid repeated code but I suppose for a small function like `getIllustration` it isn't that bad. 

An incorrect illustration name will just result in no svg rendered in the DOM. There are no errors.